src/platform/unix.rs

```diff
@@ -1,13 +1,11 @@
 #![cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "netbsd", target_os = "openbsd"))]
 
+#[cfg(feature = "wayland")]
+use smithay_client_toolkit::window::{ButtonState as SCTKButtonState, Theme as SCTKTheme};
 use std::os::raw;
```

Comment:
The import of `smithay_client_toolkit` was fine. The idea was to group `std` imports, so move that one back.